src/platform/silabs/wifi/SiWx/ncp/efx32_ncp_host.c

```diff
@@ -22,14 +22,17 @@
 #include "em_gpio.h"
 #include "em_usart.h"
 #include "gpiointerrupt.h"
+#include "sl_board_configuration_SiWx917.h"

```

Comment:
it must stay `#include <platform/silabs/wifi/SiWx/ncp/sl_board_configuration.h>`  (old line 32)and not this `sl_board_configuration_SiWx917.h`